utils/rpms/daos.spec

```diff
@@ -168,18 +170,23 @@ Requires: mercury >= %{mercury_version}
 Requires: libfabric >= %{libfabric_version}, libfabric < %{libfabric_max_version}
 %if (0%{?rhel} >= 8)
 Requires: fuse3 >= 3
+Requires: capstone >= 4.0.1
 %else
 Requires: fuse3 >= 3.4.2
+Requires: capstone >= 4.0.1
```

Comment:
This seems to be the same as the other side of the `%else`.  Surely that means it belongs outside of the `%if` altogether, no?  If it's necessary at all, per the previous comment.  So likely:

```suggestion
```

Reply:
I removed capstone. Thank you!